src/app/proposals/hooks/useGetProposalSnapshot.ts

```diff
@@ -10,5 +10,5 @@ export const useGetProposalSnapshot = (proposalId: string) => {
     args: [BigInt(proposalId)],
   })
 
-  return data
+  return BigInt(data ?? 0n)
```

Comment:
```suggestion
  return data ?? 0n
```
Or maybe this validation where the snapshot is use.

Reply:
data may be defined but is not necessarily bigint in runtime